pysgrid/sgrid.py

```diff
@@ -15,23 +15,42 @@ class SGrid(object):
                       'high': (None, 1)
                       }
     
-    def __init__(self, nodes=None, centers=None, faces=None, 
-                 edges=None, node_padding=None, face_padding=None,
-                 edge_1_padding=None, edge_2_padding=None,
-                 vertical_padding=None, grid_topology_vars=None,
-                 grid_cell_center_vars=None, grid_times=None, 
-                 variables=None, dimensions=None, face_coordinates=None,
-                 node_coordinates=None, edge_1_coordinates=None,
-                 edge_2_coordinates=None, angles=None,
-                 node_dim=None, face_dim=None,
-                 vertical_dim=None, edge_1_dim=None,
-                 edge_2_dim=None, grid_variables=None):
+    def __init__(self, nodes=None, 
+                 centers=None, 
+                 faces=None, 
+                 edges=None, 
+                 node_padding=None, 
+                 face_padding=None,
+                 volume_padding=None,
+                 edge_1_padding=None, 
+                 edge_2_padding=None,
+                 vertical_padding=None, 
+                 grid_topology_vars=None, 
+                 topology_dimension=None, 
+                 grid_cell_center_vars=None, 
+                 grid_times=None, 
+                 variables=None, 
+                 dimensions=None, 
+                 face_coordinates=None,
+                 volume_coordinates=None,
+                 node_coordinates=None, 
+                 edge_1_coordinates=None,
+                 edge_2_coordinates=None, 
+                 angles=None,
+                 node_dim=None, 
+                 face_dim=None,
+                 volume_dim=None,
+                 vertical_dim=None, 
+                 edge_1_dim=None,
+                 edge_2_dim=None, 
+                 grid_variables=None):
```

Comment:
That is a lot of variables to initialize!  Are these populated automatically when `pysgrid` reads a netcdf file?  And/or do you intend for users to manually initialize such a class?  If the former maybe we can just skip this and create them all automatically as the file is read.  If the latter I am afraid that is a very complex API.